examples/notebooks/intro/dpk_intro_1_python.ipynb

```diff
@@ -118,9 +118,9 @@
    "source": [
     "if RUNNING_IN_COLAB:\n",
     "    !mkdir -p 'input/solar-system'\n",
-    "    !wget -O 'input/solar-system/earth.pdf'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/input/solar-system/earth.pdf'\n",
-    "    !wget -O 'input/solar-system/mars.pdf'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/input/solar-system/mars.pdf'\n",
-    "    !wget -O 'my_utils.py'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/my_utils.py'"
+    "    !wget -O 'input/solar-system/earth.pdf'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/input/solar-system/earth.pdf'\n",
+    "    !wget -O 'input/solar-system/mars.pdf'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/input/solar-system/mars.pdf'\n",
+    "    !wget -O 'my_utils.py'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/my_utils.py'"
```

Comment:
Why is this here? Why isn't that in the git repo? What am I missing ?

Reply:
The .py file is in the repo, but to run on Google Colab, Sujee puts it in raw.githubusercontent.com. @sujee Can we take this file from the repo itself (but after merge into dev), as: https://github.com/IBM/data-prep-kit/blob/dev/examples/notebooks/intro/my_utils.py?

Reply:
yes, on colab, we need to download any files besides the notebook.
These are being downloaded from the official repo location